clang/include/clang/Lex/Lexer.h

```diff
@@ -136,6 +136,8 @@ class Lexer : public PreprocessorLexer {
 
   bool IsAtPhysicalStartOfLine;
 
+  bool IsCurrentLexingTokAtPhysicalStartOfLine;
```

Comment:
we have IsAtStartOfLine and IsAtPhysicalStartOfLine - Why do we need a third one?

Reply:
Thanks for your review! This change introduced by https://github.com/llvm/llvm-project/pull/102135/commits/829b1c1296148c6ca201a0d7d7951cc762d322ed. It's used to replace the `bool TokAtPhysicalStartOfLine` agurments in https://github.com/llvm/llvm-project/blob/fcefe957ddfdc5a2fe9463757b597635e3436e01/clang/lib/Lex/Lexer.cpp#L3715 , it's a bit different with `Lexer::IsAtPhysicalStartOfLine `, It represents the current token starts at the physical line. We may skip the '\n' or comment before the real token starts. with this change, we can avoid pass this value as a function argument.